### ISSUE TITLE
The docker postgres database container stops running after a time.

### ISSUE THREAD

**Issue by jlinoff** [eSentire/grape#2]
The docker postgres database container works fine for some time but stops running after awhile. It is not clear how long it runs before stopping but it is more than an hour.

I suspect that the problem is related to the /tmp or /var/lib/postgresql/data volume mounts. Another possibility is that it could be related to the system going to sleep.

**Comment by jlinoff** [comment]
I have a fix in hand but it needs to run for 8h to confirm.

This is how i am checking the time:
```bash
% gdate -u -d @$(echo "$(gdate -u +%s) - $(gdate --date $(docker inspect -f '{{ .State.StartedAt }}' demo01pg) +%s)" | bc) +'%T'
04:55:48
```

**Comment by jlinoff** [comment]
Fixed in v0.2.0.

**Comment by jlinoff** [comment]
Provided workaround that allows the container to be started without a loss of data.
